docs/modules/release-notes/pages/all-releases/4_16_1S.adoc

```diff
@@ -0,0 +1,188 @@
+
+== Standalone/On-Prem Lightning: update `adb` version
+
+This release upgraded `adb` to version 35.0.2 on the Mac mini and GEM.
+
+* For the Mac mini, the `adb` service is automatically updated when you upgrade to deviceConnect 4.16.
+* For our Standalone/On-Prem Lightning customers, please contact Kobiton Support to update the Kobiton software and `adb` on your GEM.
+
+== New virtualUSB client
+
+New virtualUSB clients for macOS and Windows are available with this release. They provide support for Android Studio Jellyfish and address an issue with blocked connections with unpaired iOS/iPadOS devices.
+
+Standalone customers can contact Kobiton support for the latest download links.
+
+[NOTE]
+====
+* iOS/iPadOS devices is not supported by virtualUSB for Windows.
+* iOS/iPadOS 17 and higher is not currently supported by virtualUSB.
+* If the device that is connected to virtualUSB cannot be released after disconnecting, quit the virtualUSB app to release it.
+====
+
+== Improvements to Scriptless Automation
+
+This release introduced several changes to the Scriptless automation workflow:
+
+* *Real-time tracking for test run statuses*, improving the experience for users monitoring in-progress test runs.
+* *Ability to view a live Scriptless Automation session* via a xref:manual-testing:start-a-mixed-session.adoc[mixed session].
++
+[NOTE]
+Avoid performing manual actions on the device during the mixed session, as they can block the running Scriptless flow.
+
+* Improved Test Management remediation flow adds more details for remediation, including an *XML Tree* to help select the correct element.
+
+== Minor improvements and bug fixes
+
+This release includes several enhancements and bug fixes to improve your day-to-day testing.
+
+* Addressed the following Script-based test automation items:
+
+** Fixed issue in XIUM where  `'-ios predicate string'` locators were not performing auto-scrolling.
+** Improved support for running more than 60 basic Appium 2 sessions on the same device.
+
+* Other items addressed:
+
+**  Fixed issue where some colors were not the same when using lightning mode.
+** Addressed an issue with modifying an existing cleanup policy.
+** Addressed an issue with Developer Disk Image not found.
+** Fixed issues related to logging into iCloud via Kobiton Apple Security Enablement prompt and with Apple ID's 2FA 6 digits field disappearing.
+** Improved last mile speed test functionality.
+** Fixed an issue with the Standalone Portal URL under SSO settings not updating after changing the Portal server's SSL certificate.
+** Fixed an issue with device running Android 15 not responding in a manual session if Gmail or Chrome is open in the background.
+
+== Standalone known issues/limitations
+
+* If Lightning mode is not turning on for a device, restart the device and try again.
+
+* The flexCorrect Appium capabilitity is currently not supported for Standalone.
```

Comment:
Appium Self-healing (flexCorrect) is not currently supported in Standalone.

Reply:
Please also add that note here: https://docs.kobiton.com/automation-testing/capabilities/available-capabilities#_flexCorrect